### PR TITLE
release 2.0.1

### PR DESCRIPTION
**Pull request by bastelfreak**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,9 @@
+## 2016/07/09 - Release 2.0.1
+
+* Fix some typos/spelling
+* modulesync with latest voxpupli defaults
```
Comment (alexjfisher) [inline CHANGELOG.md]: s/voxpupli/voxpupuli

**Comment by bastelfreak** [comment]
we've to wait with this release until https://github.com/voxpupuli/puppet-community_kickstarts/pull/13 is solved :cry:

**Comment by jyaworski** [comment]
Ping @bastelfreak

**Comment by bastelfreak** [comment]
ah

**Comment by ffrank** [comment]
:+1:

**Pull request merged**